app/0.12.x/admin-api.md

```diff
@@ -167,9 +167,9 @@ HTTP 200 OK
 
 ### Retrieve node status
 
-Retrieve usage information about a node, with some basic information about the connections being processed by the underlying nginx process, and the status of the database connection.
+Retrieve usage information about a node, with some basic information about the connections being processed by the underlying Nginx process, and the status of the database connection.
```

Comment:
We try to follow the advice from NGINX (the company) when spelling nginx (the product) - see https://forum.nginx.org/read.php?2,234083,234085#msg-234085

Reply:
Got it, reverted changes